dutch/over-het-hek/sleutel/wereld/welkeWereld.md

```diff
@@ -1,3 +1,5 @@
 Asjemenou, terwijl je dit gebeuren overdenkt vergaat de wereld onder je voeten en wat verschijnt daar?
 
-Sterren! Waar je ook kijkt zie je sterren in alle kleuren, geuren en maten.
\ No newline at end of file
+Sterren! Waar je ook kijkt zie je sterren in alle kleuren, geuren en maten.
+
+Maar dan herinner je dat je waarschijnlijk dronken bent. Daarom geef je jezelf een [harde klap in het gezicht.](harde-klap/wakker-worden-op-straat.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.